docs/configuration/configuration.md

```diff
@@ -1468,6 +1472,27 @@ See [the Prometheus eureka-sd configuration file](/documentation/examples/promet
 for a practical example on how to set up your Eureka app and your Prometheus
 configuration.
 
+### `<uyuni_sd_config>`
+
+Uyuni SD configurations allow retrieving scrape targets from managed systems
+via [Uyuni](https://www.uyuni-project.org/) API.
```

Comment:
Should we also name the downstream project?

Reply:
Not sure, I don't think we need that.